### PR TITLE
Create Swiss SMB (PME) account Template 2019 (translated and adapted from chkmu)

### PR DESCRIPTION
**Pull request by alexchap**
Adding Swiss SMB (PME) account template, based on Sterchi
Original pull request : https://github.com/Gnucash/gnucash/pull/1400

I've created a new pull request with all the corrections and the two commits that are needed

### CODE REVIEW

**Comment by fellen** [comment]
```
> xmllint --noout data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea
data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea:32: namespace error : Namespace prefix gnc-act on title is not defined
  <gnc-act:title>
                ^
data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea:35: namespace error : Namespace prefix gnc-act on short-description is not defined
  <gnc-act:short-description>
                            ^
data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea:38: namespace error : Namespace prefix gnc-act on long-description is not defined
  <gnc-act:long-description>
                           ^
data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea:42: namespace error : Namespace prefix gnc-act on exclude-from-select-all is not defined
  <gnc-act:exclude-from-select-all>1</gnc-act:exclude-from-select-all>

```
Perhaps you should just copy the namespace section from  https://wiki.gnucash.org/wiki/Account_Hierarchy_Template#Adjust_the_Header. As it is alphabetically sorted it is easier to compare.

BTW: There is no need to start new PRs after changes. I saw, you had messed up your old branch. In that case rename it to e.g. patch-broken and create a new branch patch-1 based on current maint;
then cherry-pick and amend your useful commits.
Finally force-push it.

You should usually **rebase** your work on the current maint, _never_ **merge**.

**Comment by alexchap** [comment]
Done, it should be better now. I copied it originally from a german (de_CH) template found in a gnucash mailing list and that was my mistake.

File: data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea [hunk 1]
```diff
@@ -0,0 +1,2382 @@
...
+    <act:name>Soldes initiaux</act:name>
+    <act:id type="new">a19c505d4b414bf3f5b574104959f534</act:id>
+    <act:type>EQUITY</act:type>
+    <act:commodity>
+      <cmdty:space>ISO4217</cmdty:space>
+      <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:description>Soldes initiaux</act:description>
+    <act:parent type="new">47469471b603439c88189eca6c2e86e1</act:parent>
```
Comment (fellen) [inline data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea:1676]: From my understanding it belongs under "Capitaux propres", which you named `Capitaux propres (personnes morales)`.
Is `personnes morales` really part of the name or would it better fit into 
`<act:description>Personnes morales</act:description>`?

I just saw, we didn't mention `<act:description>` and added the `List of useful columns` there.

Comment (alexchap) [reply data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea]: OK for me, thanks for the suggestion.

**Comment by fellen** [comment]
Thanks!

**Pull request merged**